exercises/perfect-numbers/perfect_numbers_test.exs

```diff
@@ -0,0 +1,76 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("perfect_numbers.exs", __DIR__)
+end
+
+ExUnit.start
+ExUnit.configure exclude: :pending, trace: true
+
+defmodule PerfectNumbersTest do
+  use ExUnit.Case
+
+  #@tag :pending
+  test "Smallest perfect number is classified correctly" do
```

Comment:
Can you wrap the chunks of tests from the canonical specs in corresponding `describe`s? Thanks